excavator/utils.py

```diff
@@ -80,7 +81,48 @@ def env_int(name, required=False, default=empty):
     value = get_env_value(name, required=required, default=default)
     if value is empty:
         raise ValueError(
-            "`env_int` requires either a default value to be specified, or, for "
+            "`env_int` requires either a default value to be specified, or for "
             "the variable to be present in the environment"
         )
     return int(value)
+
+
+def env_timestamp(name, default=empty, required=False):
+    if required and default is not empty:
+        raise ValueError("Using `default` with `required=True` is invalid")
+
+    value = get_env_value(name, required=required, default=empty)
+    # change datetime.datetime to time, return time.struct_time type
+    if default is not empty and value is empty:
+        return default
+    if value is empty:
+        raise ValueError(
+            "`env_timestamp` requires either a default value to be specified, or "
+            "for the variable to be present in the environment"
+        )
+
+    timestamp = float(value)
+    return datetime.datetime.fromtimestamp(timestamp)
```

Comment:
I assume this will be localtime and not UTC by default? Could mess with some apps if forced into UTC. Could be another parameter (utc=True).  If `True add `datetime.utcnow()` logic.